### ISSUE TITLE
multi languages support

### ISSUE THREAD

**Issue by CC2130** [rust-cli/book#69]
I wonder if the book intends to support multi languages? I've translated it into chinese, and want to share with others.

**Comment by epage** [comment]
I think bringing in more of an audience is a great idea!

Personally, I'm unfamiliar with how this works out in terms of future edits.  What kind of process generally works for that?

**Comment by CC2130** [comment]
Actually, I'm confused by this problem too. 
Maybe create a new language branch? Everytime the master is ready to release, merge from the master first, then create a translate issue, wait for someone to adopt it?

Or just show the translated version books' url in the readme and the `Getting started` page?(should contains when, from which master branch commit)

**Comment by CC2130** [comment]
> Actually, I'm confused by this problem too.
> Maybe create a new language branch? Everytime the master is ready to release, merge from the master first, then create a translate issue, wait for someone to adopt it?
> 
> Or just show the translated version books' url in the readme and the `Getting started` page?(should contains when, from which master branch commit)

zh_CN: [Rust 中的命令行工具](https://suibianxiedianer.github.io/rust-cli-book-zh_CN/)

**Comment by IsraelBuitronD** [comment]
I have cloned this repository and I have started to translate this book to spanish (`es_MX`). How can I merge my translation with this official repository in order to contribute a multi-language book?

**Comment by CC2130** [comment]
> I have cloned this repository and I have started to translate this book to spanish (`es_MX`). How can I merge my translation with this official repository in order to contribute a multi-language book?

I did this #82

**Comment by epage** [comment]
Thanks for the work on translations!  Good idea in starting small with links to unofficial repos.

**Comment by adsick** [comment]
Hi, I'm thinking on translating this book into russian but I don't want to bother hosting the page myself, so how do I do that using mdbook? upd: just looked at their repo and seems like there is no support for multilang books(

**Comment by epage** [comment]
Hosting isn't too bad if you use github.  You can create a new github org, fork this project into it, and then you can use the existing CI process to push the generated pages to github's static file server.

**Comment by CC2130** [comment]
yeah, the simplest way is that, just do your translate on your forked master branch, it will automatic create the book and push it to your github pages by SUMMARY.md, when your work is done, fork it again, just edit README.md, add your github pages links into the `mutli language support`

**Comment by adsick** [comment]
Wow, thanks, didn't know that!
